lib/resources.rb

```diff
@@ -877,259 +877,390 @@ def get_resource(key)
     # MAY 2015 BALLOT JSON
 
     MAY2015_OPEN_METADATA ||= {
-      "resourceType"=>"Conformance",
-      "text"=>{"status"=>"generated",
-               "div"=>"Generated Conformance Statement"},
-      "url"=>"https://fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Conformance",
-      "name"=>"SMART on FHIR Conformance Statement",
-      "publisher"=>"Cerner",
-      "description"=>"Describes capabilities of this server",
-      "status"=>"draft",
-      "date"=>"2015-05-06T16:03:55.188+00:00",
-      "fhirVersion"=>"0.5.0",
-      "acceptUnknown"=>false,
-      "format"=>["json"],
-      "rest"=>[{"mode"=>"server",
-                "documentation"=>"All the functionality defined in FHIR",
-                "security"=>{"cors"=>true},
-                "resource" => [
-                  {
-                    "type"=> "AllergyIntolerance",
-                    "interaction"=> [{"code"=> "search-type"
-                      }
-                    ],
-                    "searchParam"=> [
-                      {
-                        "name"=> "patient",
-                        "type"=> "reference",
-                        "documentation"=> "Who the sensitivity is for"
-                      }
-                    ]
-                  },
-                  {
-                    "type"=> "Encounter",
-                    "interaction"=> [
-                      {
-                        "code"=> "read"
-                      },
-                      {
-                        "code"=> "search-type"
-                      }
-                    ],
-                    "readHistory"=> false,
-                    "searchParam"=> [
-                      {
-                        "name"=> "patient",
-                        "type"=> "reference",
-                        "documentation"=> "The patient present at the encounter"
-                      }
-                    ]
-                  },
-                  {
-                    "type"=> "Immunization",
-                    "interaction"=> [
-                      {
-                        "code"=> "search-type"
-                      }
-                    ],
-                    "searchParam"=> [
-                      {
-                        "name"=> "patient",
-                        "type"=> "reference",
-                        "documentation"=> "The patient for the vaccination record"
+        "resourceType"=> "Conformance",
+        "text"=> {
+            "status"=> "generated",
+            "div"=> "\u003cdiv\u003eGenerated Conformance Statement\u003c/div\u003e"
+        },
+        "url"=> "https=>//fhir.sandboxcernerpowerchart.com/may2015/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Conformance",
```

Comment:
There's a find/replace issue with the protocol on some of these urls (e.g. "https=>//fhir....").